.circleci/config.yml

```diff
@@ -9,7 +12,7 @@ executors:
       - image: circleci/php:7.3
   php7_2:
     docker:
-      - image: circleci/php:7.2
+      - image: circleci/php:7.2.33
```

Comment:
To fix the major version of Xdebug run by CircleCI to 2, running Xdebug v3 with PHP7.2 without modifying php.ini will cause an error.